src/AutoRest.CSharp/AutoRest.CSharp.csproj

```diff
@@ -61,15 +61,15 @@
     </PropertyGroup>
 
     <ItemGroup>
-      <_GeneratedPropsFileLines Include="&lt;Project>" />      
```

Comment:
why is this needed?

Reply:
The changes were done by VS for proper XML escaping formatting. Let's keep it as other folks's save probably would have trigger this later anyway.